tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 distshare = {homedir}/.tox/distshare
-envlist = py38-pytest{46,50,51,52,53,54,60,61,62,70,71,latest,main},
+envlist = py38-pytest{60,61,62,70,71,latest,main},
```

Can remove `pytest60` and `61`:

```suggestion
envlist = py38-pytest{62,70,71,latest,main},
```

---

sure :+1: